src/REPLMode/command_declarations.jl

```diff
@@ -103,7 +103,7 @@ that matches, or ranges thereof, such as `@1.2-3.4.5`. A git revision can be
 specified by `#branch` or `#commit`.
 
 If a local path is used as an argument to `add`, the path needs to be a git repository.
-The project will then track that git repository just like it would track a remote repository online.
+The project will then track that git repository just like it would track a remote repository online. If the package is not located at the top of the git repository, a subdirectory can be specified with `path/[subdir/path]`.
```

Comment:
I am not sure if this project has any conventions for this, but perhaps avoid overlong lines?
They are annoying to read in the web UI and in diffs, and the rest of the comments seems to
avoid them too?
```suggestion
The project will then track that git repository just like it would track a remote repository online.
 If the package is not located at the top of the git repository, a subdirectory can be specified
 with `path/[subdir/path]`.
```